home/ivp-planning/advanced-tools/path-tool.md

```diff
@@ -1,102 +1,80 @@
+![](../../../.gitbook/assets/iVP_path_tool_mouse_icon.jpg)
 
-To create a path, **left click on a position** in the [2D-panel](../user-interface/the-2d-panel.md). A small circle will appear that marks the spot of the click and functions as first measurement point. When the mouse is moved after the click, a line will appear between the measurement point and the mouse pointer. To set a new measurement point it is sufficient to simply **left click on another position** in the [2D-panel](../user-interface/the-2d-panel.md).
+To create a path, **left-click on a position** in the [2D-panel](../user-interface/the-2d-panel.md). A small circle will appear, marking the spot of the click and functioning as the first measurement point. When the mouse is moved after the click, a line will appear between the measurement point and the mouse pointer. To set a new measurement point, simply **left-click on another position** in the [2D-panel](../user-interface/the-2d-panel.md).
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_line.jpg)
+![](../../../.gitbook/assets/iVP_path_tool_draw_path_line_new.jpg)
 
-During the creation process of the path, every line and every corner will automatically display its length or angle.
+During the creation process of the path, each line and corner will automatically display its length or angle.
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_angle.jpg)
+![](../../../.gitbook/assets/iVP_path_tool_daw_path_angle_new.jpg)
 
-To complete a path it is either possible to press **Enter** or **double click on the last measurement point** set. With its completion, the path will also be saved as a permanent object that can be adjusted or deleted.
+To complete a path, you can either press **Enter** or **double-click on the last measurement point** set. Upon completion, the path is saved as a permanent object that can be adjusted or deleted in the [hierarchy panel](../user-interface/hierarchy-panel.md)
 
 {% hint style="info" %}
 If the path contains three or more measurement points, it will automatically be closed after creation (the first and the last measurement points will be connected automatically).
 {% endhint %}
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_path\_panel.jpg)
+![](../../../.gitbook/assets/iVP_path_tool_draw_path_panel_new.jpg)
 
-The path creation can be cancelled at any time by pressing the **Escape key**. This can be used to quickly measure distances or angles without creating a permanent path.
+The path creation can be canceled at any time by pressing the **Escape key**. This allows for quick measurement of distances or angles without creating a permanent path.
 
-## Path selection and editing:
+## Path selection and editing
 
-Existing paths can be selected by **clicking on the path name** in the [2D-panel](../user-interface/the-2d-panel.md).
+Existing paths can be selected by **clicking on the path name** in the [hierarchy panel](../user-interface/hierarchy-panel.md).
```

Comment:
Can also be selected from the 2D- and 3D view like any other object. I think we don't need the extra section about selection.